### PR TITLE
Grafana updates : verified on ubuntu 16

### PR DESCRIPTION
**Pull request by sid226**
LOGS [grafana-5.2.4-2018-10-17-10%3A44%3A48.log](https://github.com/imdurgadas/scripts/files/2487082/grafana-5.2.4-2018-10-17-10.3A44.3A48.log)

### CODE REVIEW

**Comment by sid226** [comment]
Verified on RHEL 7.3 [grafana-5.2.4-2018-10-17-13%3A19%3A45.log](https://github.com/imdurgadas/scripts/files/2487684/grafana-5.2.4-2018-10-17-13.3A19.3A45.log)

**Comment by sid226** [comment]
Added Protobuf  script : verified on UBUNTU 16
log for ubuntu 16 [protobuf-3.6.1-2018-10-18-05%3A30%3A58.log](https://github.com/imdurgadas/scripts/files/2490448/protobuf-3.6.1-2018-10-18-05.3A30.3A58.log)

**Comment by sid226** [comment]
Verified Protobuf on UBUNTU18, SLES 15.12.3 , RHEL 7.4,7.5 

log for UBUNTU18
[protobuf-3.6.1-2018-10-18-06%3A11%3A26.log](https://github.com/imdurgadas/scripts/files/2490966/protobuf-3.6.1-2018-10-18-06.3A11.3A26.log)

log for SLES 15.12.3 
[protobuf-3.6.1-2018-10-18-05%3A50%3A38.log](https://github.com/imdurgadas/scripts/files/2490971/protobuf-3.6.1-2018-10-18-05.3A50.3A38.log)
[protobuf-3.6.1-2018-10-18-05%3A50%3A09.log](https://github.com/imdurgadas/scripts/files/2490972/protobuf-3.6.1-2018-10-18-05.3A50.3A09.log)
log for RHEL 7.4,7.5
[protobuf-3.6.1-2018-10-18-07%3A15%3A21.log](https://github.com/imdurgadas/scripts/files/2490981/protobuf-3.6.1-2018-10-18-07.3A15.3A21.log)
[protobuf-3.6.1-2018-10-18-07%3A15%3A29.log](https://github.com/imdurgadas/scripts/files/2490983/protobuf-3.6.1-2018-10-18-07.3A15.3A29.log)

**Comment by sid226** [comment]
Verified Protobuf for RHEL 6.10  
[protobuf-3.6.1-2018-10-18-04%3A53%3A17.log](https://github.com/imdurgadas/scripts/files/2491552/protobuf-3.6.1-2018-10-18-04.3A53.3A17.log)

File: cAdvisor/build.sh [hunk 1]
```diff
@@ -147,12 +165,12 @@ function logDetails() {
...
 	echo
 	echo "Usage: "
-	echo "  install.sh  [-d <debug>] [-v package-version] [-y install-without-confirmation]"
+	echo "  install.sh  [-d <debug>] [-v package-version] [-y install-without-confirmation] [-t install-with-tests]"
 	echo "       default: If no -v specified, latest version will be installed"
 	echo
 }
 
-while getopts "h?dyv:" opt; do
+while getopts "h?dyv:t" opt; do
```
Comment (imdurgadas) [inline cAdvisor/build.sh]: Place t before v:
```suggestion
while getopts "h?dytv:" opt; do
```

**Pull request merged**